src/main.rs

```diff
@@ -101,6 +116,9 @@ fn print_help_section() {
 
     USAGE:
       add                               Starts a prompt to add an coauthor.
+
```

Comment:
I think it would look better without this newline.

Reply:
I was afraid that the text from the section above would get mixed up. Maybe not a problem. I'll change